build/images/Dockerfile.build.windows

```diff
@@ -1,29 +1,23 @@
 ARG GO_VERSION
-FROM mcr.microsoft.com/powershell:lts-nanoserver-1809 as cni-binaries-windows
+FROM gavinxin/lib-windows:20230324 as windows-download
 SHELL ["pwsh", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ENV CNI_PLUGINS="./host-local.exe"
 
 WORKDIR /opt/cni/bin
```

Comment:
If you don't download host-local.exe in this intermediate container, you don't need to switch the workdir here.